Header header

# references
float64 x_ref
float64 y_ref
float64 z_ref
float64 heading_ref

# state
geometry_msgs/Vector3 position_W
geometry_msgs/Vector3 velocity_W
geometry_msgs/Vector3 euler_angle
geometry_msgs/Vector3 euler_rate

# virtual control
float64 total_thrust
geometry_msgs/Vector3 moment
